mixmatch/proxy.py

```diff
@@ -169,13 +195,13 @@ def _search_forward(self):
         for sp in CONF.proxy.service_providers:
             if sp == 'default':
                 response = self._do_request_on('default')
-                if 200 <= response.status_code < 300:
+                if 200 <= response.status_code <= 300:
```

Comment:
that is needed *only if* want to forward the call with no action to sp's, and for volume api, calls with no action return 300. BUT we are not following this approach anymore, so this has been fixed.